sdk/features/arm-features/tsconfig.json

```diff
@@ -3,13 +3,14 @@
     "module": "es6",
     "moduleResolution": "node",
     "strict": true,
-    "target": "es5",
+    "target": "es6",
     "sourceMap": true,
     "declarationMap": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es6"],
+    "preserveConstEnums": true,
+    "lib": ["es6", "dom"],
```

Comment:
why adding dom here?

Reply:
@sarangan12

Reply:
In one of the data plane SDKs, this was creating issue. I do not remember which SDK. On analysis, I found this was generic and we should add the dom also. If you feel this is not required specifically for this SDK, you can remove it if there are no issues on your side.

Reply:
@witemple-msft and @xirzec what do you think about this?

Reply:
Better to omit it if your SDK doesn't only work in browsers, but sometimes it's hard to avoid. I'd need to see the build error.

Reply:
It should be fine to include as long as we're careful to shim browser types that are exposed in the API surface. I took a quick look through the api.md and didn't see anything that sticks out to me.

Reply:
This is emitted by code gen (@sarangan12  for all packages, right?) so I think we're careful in the sense that code gen does not understand browser types/shims anyway.